server/audit_math/supersimple.py

```diff
@@ -15,6 +15,17 @@
 o2: float = 0.0001
 u2: float = 0.0001
 
+# { ballot_id: { contest_id: { choice_id: 0 | 1 }}}
+CVRS = Dict[str, Dict[str, Dict[str, int]]]
```

Comment:
It's better as an integer because it lets us do math with the votes without having to cast. Also, in the future if we support RCV then it will be necessary.